src/.nuget/dir.props

```diff
@@ -34,6 +34,7 @@
   <!-- derive an OS Group based on the OS Family -->
   <PropertyGroup>
     <_derivedPackageTargetOSGroup Condition="'$(_derivedPackageTargetOSGroup)' == '' and '$(_runtimeOSFamily)' == 'osx'">OSX</_derivedPackageTargetOSGroup>
+    <_derivedPackageTargetOSGroup Condition="'$(_derivedPackageTargetOSGroup)' == '' and '$(_runtimeOSFamily)' == 'android'">Android</_derivedPackageTargetOSGroup>
```

Comment:
@chcosta FYI, Implementing the above causes the package build for `Microsoft.NETCore.Runtime.CoreCLR` to fail. Specifically the build ignores the conditional I've copied from the `runtime.Android` file to the `runtime.Linux`file  to exclude `System.Private.Corelib.Ni.dll` from the package. If I use the existing condition for Ni  it does work…but since `Ni` files can be generated post-build on Android it shouldn't be used.